packages/codeeditor/src/editor.ts

```diff
@@ -568,6 +598,11 @@ export namespace CodeEditor {
      * Gets the list of tokens for the editor model.
      */
     getTokens(): IToken[];
+
+    /**
+     * Finds definitions of variable or function matching given name.
+     */
+    findDefinitions(name: string): Array<IToken>;
```

Comment:
I think this should probably be promise-based in order to allow for the possibility of checking with a server-side component for definitions.